operations/producer/producer-operation.go

```diff
@@ -91,25 +119,33 @@ func (operation *ProducerOperation) Produce(topic string, flags ProducerFlags) {
 	if flags.Value != "" {
 		value = []byte(flags.Value)
 	} else if stdinAvailable() {
-		bytes, err := ioutil.ReadAll(os.Stdin)
-		if err != nil {
-			output.Failf("Failed to read data from the standard input: %s", err)
-		}
+		scanner := bufio.NewScanner(os.Stdin)
+		for scanner.Scan() {
+			bytes, err := scanner.Text(), scanner.Err()
+			if err != nil {
+				output.Failf("Failed to read data from the standard input: %s", err)
+			}
 
-		if len(bytes) > 0 && bytes[len(bytes)-1] == 10 {
-			// remove trailing return
-			bytes = bytes[:len(bytes)-1]
-		}
+			if len(bytes) > 0 && bytes[len(bytes)-1] == 10 {
```

Comment:
this if block can be removed since `scanner.Scan()` will already strip the trailing newline